stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -63,8 +65,19 @@ public class AttachmentGallery : ConstraintLayout {
         configureImagesMenu()
     }
 
-    public fun provideImageList(fragmentActivity: FragmentActivity, imageList: List<String>, currentIndex: Int = 0) {
-        adapter = AttachmentSlidePagerAdapter(fragmentActivity, imageList)
+    public fun provideImageList(
+        fragmentActivity: FragmentActivity,
+        imageList: List<String>,
+        currentIndex: Int = 0,
+        imageClickListener: () -> Unit = {},
+    ) {
+        adapter = AttachmentSlidePagerAdapter(fragmentActivity, imageList) {
+            binding.bottomBarGroup.isVisible = isFullScreen
```

Comment:
This line doesn't seem to make sense - the bottom group is visible when we're _not_ full screen, isn't it? I think we should flip the flag first, and _then_ update this visibility, e.g.

```kotlin
isFullScreen = !isFullScreen
binding.bottomBarGroup.isVisible = !isFullScreen
```